lib/drafter.js

```diff
@@ -4,12 +4,18 @@ try {
   module.exports = {
     parse: protagonist.parse,
     parseSync: protagonist.parseSync,
+
+    validate: validate,
```

Comment:
Ah, I had that originally. I wrote some stub validate functions to test my tests. Forgot to restore this back before committing. Updated.